### PR TITLE
feat(header) : implement header structure

### PR DESCRIPTION
**Pull request by kibo-susantasamanta**
@kibo-sushant @kibo-kevinwatts @amolpawar-kibo

### CODE REVIEW

File: components/LoginModal.vue [hunk 1]
```diff
@@ -32,14 +32,15 @@
             label="Remember me"
             class="form__element form__checkbox"
           />
-          <span v-if="userError.login" style="color: red">
+          <span v-if="userError.login" style="color: red; display: flex; justify-content: center">
```
Comment (kibo-sushant) [inline components/LoginModal.vue]: Put CSS in CSS section instaed of writing it online

Comment (kibo-susantasamanta) [reply components/LoginModal.vue]: Resolved

File: components/KiboAppHeader.vue [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+import { SfImage, SfIcon, SfButton, SfSearchBar, SfMenuItem, SfLink } from "@storefront-ui/vue"
+import { computed, ref, onBeforeUnmount, defineComponent } from "@vue/composition-api"
+import { clickOutside } from "@storefront-ui/vue/src/utilities/directives/click-outside/click-outside-directive.js"
+import {
+  mapMobileObserver,
+  unMapMobileObserver,
+} from "@storefront-ui/vue/src/utilities/mobile-observer.js"
+
+import { useAsync } from "@nuxtjs/composition-api"
+import useUiState from "../composables/useUiState"
```
Comment (kibo-kevinwatts) [inline components/KiboAppHeader.vue:78]: stay consistent, use `@/composables/useUIState`

**Review by kibo-kevinwatts — APPROVED** [review]

File: components/KiboAppHeader.vue (hunk 2 above)
Comment (kibo-susantasamanta) [reply components/KiboAppHeader.vue:78]: Resolved

**Pull request merged**